Modules/SubModule_Import/xASL_imp_DCM2NII_Subject_ShuffleTheDynamics.m

```diff
@@ -1,9 +1,10 @@
-function [nii_files, summary_line, globalCounts, ASLContext] = xASL_imp_DCM2NII_Subject_ShuffleTheDynamics(globalCounts, scanpath, scan_name, nii_files, iSubject, iSession, iScan)
+function [x,nii_files, summary_line, globalCounts, ASLContext] = xASL_imp_DCM2NII_Subject_ShuffleTheDynamics(x,globalCounts, scanpath, scan_name, nii_files, iSubject, iSession, iScan)
 %xASL_imp_DCM2NII_Subject_ShuffleTheDynamics Shuffle the dynamics.
 %
 % FORMAT: [nii_files, summary_line, globalCounts, ASLContext] = xASL_imp_DCM2NII_Subject_ShuffleTheDynamics(globalCounts, scanpath, scan_name, nii_files, iSubject, iSession, iScan)
 % 
 % INPUT:
+%   x               - ExploreASL x struct (STRUCT, REQUIRED)
```

Comment:
We don't need `x` anymore here as input/output arguments right? Always good to check your difference before creating the PR.

Reply:
We do, for `x.dir.studyPar`.

Reply:
But @BeatrizPadrela could update the `% FORMAT. ...` line

Reply:
will do!